modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc

```diff
@@ -598,6 +600,27 @@ be ignored. This flag depends on the `wnd.activity.registerExtension` and has th
 'wnd.activity.sendToSharees' => false,
 ....
 
+=== Make the group membership component to assume the ACL contains a user
+The WND app doesn't have knowledge about the users or groups associated to ACLs. This
+means that an ACL containing "admin" might refer to a user called "admin" or a
+group called "admin". By default, the group membership component consider the ACLs to
+target groups, and as such, it will try to get the information of such group. This
+works fine if the majority of the ACLs target groups. If the majority of the ACLs
+contains users, this might be problematic. The cost of getting the information of a
+group is usually higher than getting the information of a user. As such, this option
```

Comment:
```suggestion
contains users, this might be problematic. The cost of getting information on a
group is usually higher than getting information on a user. As such, this option
```